cli/pkg/providers/minikube/minikube.go

```diff
@@ -45,114 +43,49 @@ func (p *provider) GetOverrides() template.Variables {
 }
 
 func (p *provider) CheckEnvironment() error {
-	return workspace.CheckBinaries([]string{"minikube"})
-}
-func (p *provider) PreInstallation(workspace string) (string, error) {
-	return workspace, nil
+	return workspace.CheckBinaries([]string{"minikube", "terraform"})
```

Comment:
I would also add a check for the docker binary as it is needed in the `local exec` resource in minikube's `main.tf`.

Reply:
As a side note - maybe to include in the docs. It is better if users install docker via the official website rather than brew. Otherwise the you get an error where the `docker deamon is not installed`.

Reply:
But I guess this depends on which driver they choose to use. i.e, they can still use VirtualBox.. Then the solution might be to capture the provider-specific variables from the CLI and then check those. This is an issue for later that we discussed.

Reply:
Good point. But I think this falls under the `minikube installation`. Perhaps we can add a note that a driver is required to run Minikube successfully, but I wouldn't check for any other binaries and drivers.

Reply:
Ok 👍 ! But if they do not have the `docker` installed correctly, for example, the terraform will fail mid-way